### DISCUSSION TITLE
Time for Code Discussion

### DISCUSSION THREAD

**Discussion by kgnet88** [mgerhold/silly-sprites#33] [General]
If we have questions or comments about the code, should we ask them once the code on a branch has been visibly checked in on a development branch, or wait until it's merged on the main branch?

The reason why I ask is the following, after your last stream I looked in the OpenGL branch and there I noticed 2 things, which is why I also asked my question in the associated issue. But when I looked back at the code later, after you opened the discussion area here, one had already been changed and the other is still in the code, which is still not merged and is still changing.

Hence my question, should I hold back my comment until the code has actually ended up in the main branch or should I make it earlier (I see merits for both positions)?

**Comment by mgerhold** [comment]
For me personally it is totally fine if you comment on any code that's publicly visible (no matter on which branch it is). If you find an issue there, it is likely that I just missed it myself. So I would appreciate any comment with the goal to improve the code.
However, I cannot speak for @PurpurTentakel97 here.
